vllm/model_executor/models/arctic.py

```diff
@@ -388,8 +390,11 @@ def forward(
         positions: torch.Tensor,
         kv_caches: List[torch.Tensor],
         attn_metadata: AttentionMetadata,
+        inputs_embeds: Optional[torch.Tensor] = None,
+        inputs_embeds_masks: Optional[torch.Tensor] = None,
```

Comment:
The argument list is getting a bit cluttered, but we can deal with this in another PR.